### PR TITLE
Document how to set up multiple event persisters

### PR DESCRIPTION
**Pull request by erikjohnston**

### CODE REVIEW

File: docs/workers.md [hunk 1]
```diff
@@ -319,6 +319,18 @@ stream_writers:
     events: event_persister1
 ```
 
+The `events` stream also experimentally supports having multiple writers, where
+work is sharded between them by room ID. Note that you *must* restart all
+instances when adding or removing event persisters. An example `stream_writers`
```
Comment (anoadragon453) [inline docs/workers.md:324]:
```suggestion
work is sharded between them by room ID. Note that you *must* restart all
even persister instances when adding or removing event persisters. An example `stream_writers`
```

While a little repetitive, I originally wasn't sure whether this meant all event persisters or all workers.

**Review by anoadragon453 — APPROVED** [review]
lgtm otherwise!

File: docs/workers.md (hunk 1 above)
Comment (erikjohnston) [reply docs/workers.md:324]: I do mean all instances though. Will try and reword

Comment (anoadragon453) [reply docs/workers.md:324]: Oh, just say "restart all worker instances" perhaps?

**Comment by richvdh** [comment]
this follows up #8294.

**Comment by richvdh** [comment]
related: #7986

**Pull request merged**